Dockerfile

```diff
@@ -60,8 +60,7 @@ RUN go install github.com/gogo/protobuf/protoc-gen-gogoslick
 RUN go install github.com/gogo/protobuf/protoc-gen-gogotypes
 RUN go install github.com/gogo/protobuf/protoc-gen-gostring
 ENV GO111MODULE=on
-RUN go get github.com/chrusty/protoc-gen-jsonschema/cmd/protoc-gen-jsonschema@switch_jsonschema_library
-RUN go install github.com/chrusty/protoc-gen-jsonschema/cmd/protoc-gen-jsonschema@switch_jsonschema_library
+RUN go install github.com/chrusty/protoc-gen-jsonschema/cmd/protoc-gen-jsonschema@1.1.0
```

Comment:
Yes, we need the `on` and `off` for the chrusty library to specify a version.  Same case for the atlas-openapiv2-patch in the lines further below this line.

But, I didn't need the `off` after atlas-openapiv2-patch since no installs are below that line and so i have removed that line.